TM1py/Services/RestService.py

```diff
@@ -236,7 +236,7 @@ def __init__(self, **kwargs):
             self._base_url = kwargs['base_url']
```

Comment:
@rclapp 
Could we do something like this to ensure backward compatibility?

``` python
self._base_url = kwargs['base_url'].strip("/api/v1") + "/api/v1"
            
```

Reply:
Not sure that will work as v12 URLs don't end with api/v1/ only v11 URLs do.

v12: http://domain.mycompany.com/MyInstance/api/v1/Databases('tm1-kgh')

We could expand TM1Service to understand "Instance" and "Database" that way if an Instance and Database was specified it would know how to insert them into the address. If they were left out then the old structure of URL would work.